keras/backend/tensorflow_backend.py

```diff
@@ -416,7 +416,7 @@ def rnn(step_function, inputs, initial_states,
 def switch(condition, then_expression, else_expression):
     '''condition: scalar tensor.
     '''
-    return tf.control_flow_ops.cond(condition,
+    return tf.python.control_flow_ops.cond(condition,
```

Comment:
Style nitpick: the lines below should be indented at the level of the parens (same with the modifications above).